lib/helpers/frontend/resolve-image-url.ts

```diff
@@ -0,0 +1,12 @@
+import { PopulatedBook } from "@lib/services/book"
+
+export function resolveImageUrl(book: PopulatedBook | null | undefined): string {
+  if (!book || !book.imageUrl) return 'https://campusstore.mcmaster.ca/cgi-mcm/ws/getTradeImage.pl?isbn=9781264595310';
```

Comment:
I think we should host a fallback image on our end, maybe something that fits the styling of our website. For now, I think that this is okay.